documentation/modules/configuring/con-config-mirrormaker2.adoc

```diff
@@ -76,97 +76,121 @@ For more information, see xref:con-high-volume-config-properties-{context}[Handl
-      config.storage.replication.factor: 1
-      offset.storage.replication.factor: 1
-      status.storage.replication.factor: 1
-    tls: # <13>
-      trustedCertificates:
-        - pattern: "*.crt"
-          secretName: my-cluster-target-cluster-ca-cert
+    - alias: "my-cluster-source" # <5>
+      # Authentication (recommended)
+      authentication: # <6>
+        certificateAndKey:
+          certificate: source.crt
+          key: source.key
+          secretName: my-user-source
+        type: tls
+      bootstrapServers: my-cluster-source-kafka-bootstrap:9092 # <7>
+      # TLS configuration (optional)
+      tls: # <8>
+        trustedCertificates:
+          - pattern: "*.crt"
+            secretName: my-cluster-source-cluster-ca-cert
+    - alias: "my-cluster-target" # <9>
+      # Authentication (recommended)
+      authentication: # <10>
+        certificateAndKey:
+          certificate: target.crt
+          key: target.key
+          secretName: my-user-target
+        type: tls
+      bootstrapServers: my-cluster-target-kafka-bootstrap:9092 # <11>
+      # Kafka Connect configuration (optional)
+      config: # <12>
+        config.storage.replication.factor: 1
+        offset.storage.replication.factor: 1
+        status.storage.replication.factor: 1
+      # TLS configuration (optional)
+      tls: # <13>
+        trustedCertificates:
+          - pattern: "*.crt"
+            secretName: my-cluster-target-cluster-ca-cert
+  # Mirroring configurations (essential)
   mirrors: # <14>
-  - sourceCluster: "my-cluster-source" # <15>
-    targetCluster: "my-cluster-target" # <16>
-    sourceConnector: # <17>
-      tasksMax: 10 # <18>
-      autoRestart: # <19>
-        enabled: true
-      config
-        replication.factor: 1 # <20>
-        offset-syncs.topic.replication.factor: 1 # <21>
-        sync.topic.acls.enabled: "false" # <22>
-        refresh.topics.interval.seconds: 60 # <23>
-        replication.policy.class: "org.apache.kafka.connect.mirror.IdentityReplicationPolicy" # <24>
-    heartbeatConnector: # <25>
-      autoRestart:
-        enabled: true
-      config:
-        heartbeats.topic.replication.factor: 1 # <26>
-        replication.policy.class: "org.apache.kafka.connect.mirror.IdentityReplicationPolicy"
-    checkpointConnector: # <27>
-      autoRestart:
-        enabled: true
-      config:
-        checkpoints.topic.replication.factor: 1 # <28>
-        refresh.groups.interval.seconds: 600 # <29>
-        sync.group.offsets.enabled: true # <30>
-        sync.group.offsets.interval.seconds: 60 # <31>
-        emit.checkpoints.interval.seconds: 60 # <32>
-        replication.policy.class: "org.apache.kafka.connect.mirror.IdentityReplicationPolicy"
-    topicsPattern: "topic1|topic2|topic3" # <33>
-    groupsPattern: "group1|group2|group3" # <34>
+    - sourceCluster: "my-cluster-source" # <15>
+      targetCluster: "my-cluster-target" # <16>
+      # Source connector configuration (optional)
```

Comment:
I would say that this is essential (or at least recommended) as MM2 makes little sense without it.